packages/terra-abstract-modal/src/_ModalContent.jsx

```diff
@@ -138,15 +138,29 @@ const ModalContent = forwardRef((props, ref) => {
         ref={ref}
       >
         <FormattedMessage id="Terra.AbstractModal.BeginModalDialog">
```

Comment:
Should we take advantage of the new fallback message logic ? Where if a translation is missing, it will fallback to the value defined here?
https://formatjs.io/docs/react-intl/components#usage

It might end up with mis-matched locales...but might be better than the translation key?

Reply:
Is the fallback logic available for all versions we support? IMO i'm ok if we don't fall back, if we're falling back there's something wrong with their app setup and we should make it obvious.

Reply:
We'll probably want to update to use that once we do next major version bumps on components, but in order to continue supporting the components to be used with the older versions of react-intl we can't.